app/views/results/_csv_export_modal_form.html.erb

```diff
@@ -0,0 +1,42 @@
+<div class="modal fade" id="exportModal" tabindex="-1" role="dialog" aria-labelledby="exportModalLabel" aria-hidden="true">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <%= form_tag(export_results_path, method: "get") do %>
+        <div class="modal-header">
+          <h5 class="modal-title">Export CSV</h5>
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+        </div>
+
+        <div class="modal-body">
+          <div class="form-group">
+            <%= label_tag :to_email %>
+            <%= text_field_tag :to_email, nil, required: true, class: "form-control" %>
+          </div>
+
+          <div class="form-group">
+            <div class="form-row">
```

Comment:
Don't think you need form row on these ones, look at the email one above

Reply:
Thanks, I will fix this.